src/tests/component.test.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import renderer from 'react-test-renderer';
+import Quotes from '../Routes/Quote';
+import Home from '../Routes/Home';
+import Calculator from '../Routes/Calculator';
+import EndRow from '../components/Calculator/EndRow';
+import ResultScreen from '../components/Calculator/ResultScreen';
+import RowCalculator from '../components/Calculator/RowCalculator';
+
```

Comment:
hey @julie-ify, I have added testing for button click in `userInteraction.js` and checked if the state is being updated or not on click of a button because that state is being  passed to the `calculate()` which is doing actual calculations and has a test of its own

Reply:
hey @julie-ify ,
Thanks for suggesting the change.
I did add the user interaction testing file inside the testing folder but by mistake named it `userInteraction.js`  instead of `userInteraction.test.js`  😀😀😀 
Let me change that and ask for another review, hopefully, will get approved this time 😊